pkg/sqlite/query.go

```diff
@@ -25,6 +25,10 @@ func NewSQLLiteQuerier(dbFilename string) (*SQLQuerier, error) {
 	return &SQLQuerier{db}, nil
 }
 
+func NewSQLLiteQuerierFromDb(db *sql.DB) (*SQLQuerier, error) {
```

Comment:
Why return an error here if it will always be `nil`?

Reply:
I honestly don't know how it got there. Changing it.